apps/zotonic_mod_development/src/mod_development.erl

```diff
@@ -250,3 +336,27 @@ observe_admin_menu(#admin_menu{}, Acc, Context) ->
 
      |Acc].
 
+exec_chrome(Executable, SiteUrl, ExtraArgs, #{secure := true} = Options) ->
+    TmpPath = filename:join([<<"/">>, <<"tmp">>, <<"foo">>]),
+    Args = lists:join(" ", [
+        "--user-data-dir=" ++ TmpPath,
+        "--ignore-certificate-errors",
+        "--unsafely-treat-insecure-origin-as-secure=" ++ SiteUrl
+        | ExtraArgs
+    ]),
+    do_exec_chrome(Executable, SiteUrl, Args, Options);
+exec_chrome(Executable, SiteUrl, ExtraArgs, Options) ->
+    do_exec_chrome(Executable, SiteUrl, ExtraArgs, Options).
+
+do_exec_chrome(Executable, SiteUrl, Args, _Options) ->
+    Command = io_lib:format("~s ~s ~s", [Executable, Args, SiteUrl]),
```

Comment:
We need to escape the path and siteUrl here as they might contain spaces and/or other characters.

Options to use are `z_filelib:os_escape(Executable)`, and `z_filelib:os_filename(SiteUrl)`